acs-engine.go

```diff
@@ -122,6 +163,8 @@ var noPrettyPrint = flag.Bool("noPrettyPrint", false, "do not pretty print outpu
 var artifactsDir = flag.String("artifacts", "", "directory where artifacts will be written")
 var classicMode = flag.Bool("classicMode", false, "enable classic parameters and outputs")
 var parametersOnly = flag.Bool("parametersOnly", false, "only output the parameters")
+var caCertificatePath = flag.String("caCertificatePath", "", "the path to the CA Certificate file")
```

Comment:
The CA secret is not to be stored in the model.  Early on there was sensitivity around this, and that is why we made it not a public property in the API model and it is not deserialized or serialized to the file.  We could revoke the other certs/key, but not the root CA.